tauri/examples/communication/src-tauri/src/main.rs

```diff
@@ -18,19 +18,21 @@ struct Context;
 
 fn main() {
   tauri::AppBuilder::<tauri::flavors::Wry, Context>::new()
-    .setup(|dispatcher, _source| async move {
-      let mut dispatcher = dispatcher.clone();
-      tauri::event::listen(String::from("js-event"), move |msg| {
+    .setup(|webview_manager| async move {
+      let current_webview = webview_manager.current_webview().unwrap().clone();
+      let current_webview_ = current_webview.clone();
```

Comment:
Is there any way to get around this quite ugly clone mess? If a function like `listen` always requires a second clone, could it clone automatically?

Reply:
No way around it, Rust is verbose :/